nvflare/fuel/f3/streaming/tools/utils.py

```diff
@@ -13,16 +13,24 @@
 # limitations under the License.
 import logging
 
+from nvflare.fuel.f3.connection import BytesAlike
 from nvflare.fuel.f3.streaming.stream_utils import wrap_view
 
 BUF_SIZE = 64 * 1024 * 1024 + 1
 TEST_CHANNEL = "stream"
 TEST_TOPIC = "test"
 TX_CELL = "sender"
-RX_CELL = "receiver"
+RX_CELL = "server"  # Passive cell's fqcn must be "server"
```

Comment:
is our fqcn always "server"?

Reply:
Cellnet is hardcoded to set the cell in listen mode if the fqcn is server. So one of the cell must use this name.